sound/soc/codecs/hdac_hdmi.c

```diff
@@ -1319,7 +1319,7 @@ static int hdac_hdmi_add_pin(struct hdac_device *hdev, hda_nid_t nid)
 	struct hdac_hdmi_pin *pin;
 	int ret;
 
-	pin = kzalloc(sizeof(*pin), GFP_KERNEL);
+	pin = devm_kzalloc(&hdev->dev, sizeof(*pin), GFP_KERNEL);
```

Comment:
Should ```ports = kcalloc(max_ports, sizeof(*ports), GFP_KERNEL); ``` in ```hdac_hdmi_add_ports()``` be the same?

Reply:
Ah yes, I removed kzalloc occurrences but not kcalloc. Good catch indeed.